operations/configuration.md

```diff
@@ -383,7 +391,31 @@ Returns all tax environments supported by the API.
 | `Code` | string | required | Code of the tax rate. |
 | `TaxationCode` | string | required | Code of the taxation. |
 | `Value` | number | required | Tax rate, e.g. `0.21` in case of 21% tax rate. |
+| `Strategy` | [Tax Rate Strategy](configuration.md#tax-rate-strategy) | required | Tax strategy type, e.g. relative or flat. |
+
+#### Tax Rate Strategy
+
+| Property | Type |  | Description |
+| --- | --- | --- | --- |
+| `Discriminator` | [Tax Strategy Discriminator](configuration.md#tax-strategy-discriminator)| required | If tax rate is flat or relative. |
+| `Value` | object | required | Based on Tax Strategy Discriminator, e.g. relative or flat. |
+
+#### Flat Tax Strategy
+
+| Property | Type |  | Description |
+| --- | --- | --- | --- |
+| `Value` | number | required | Absolute value of tax. |
+| `CurrencyCode` | string | required | Code of [Currency](configuration.md#currency). |
+
+#### Relative Rate Strategy
+
+| Property | Type |  | Description |
+| --- | --- | --- | --- |
+| `Value` | number | required | Tax rate, e.g. `0.21` in case of 21% tax rate. |
 
+#### Tax Strategy Discriminator
+* `Relative` 
+* `Flat`
```

Comment:
Misssing description of flat and relative

Reply:
@vitasamek Thank you very much for the quick feedback. I'll fix this ASAP. Can you help me understand why these two need further description when some lists like [Customer classification](https://mews-systems.gitbook.io/connector-api/operations/customers#customer-classification) for example doesn't? Doesn't relative vs. flat amount in reference to taxes count as common knowledge? If no, what is the logic I should use to decide this for future lists?

Reply:
But for example 
#### Image resize mode 
has descriptions and this is place where i guess it make sense to have short description

Reply:
@patrikmiksovsky95 I had a call with @jirrick after Vita's comment and he suggested to resolve it by adding links to each of the strategy types to the description of `Value`. 
![image](https://user-images.githubusercontent.com/49149492/81564999-eb498f00-9398-11ea-855e-0e080d42d38b.png)